vendor/GoogleCast/CastCompanionLibrary/build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Lets remove CastCompanionLibrary as we are not using it.

Reply:
done